distribution/io/src/assemble/io.xml

```diff
@@ -65,5 +65,6 @@
     <file><source>${basedir}/../../pulsar-io/netty/target/pulsar-io-netty-${project.version}.nar</source></file>
     <file><source>${basedir}/../../pulsar-io/mongo/target/pulsar-io-mongo-${project.version}.nar</source></file>
     <file><source>${basedir}/../../pulsar-io/debezium/mysql/target/pulsar-io-debezium-mysql-${project.version}.nar</source></file>
+	<file><source>${basedir}/../../pulsar-io/debezium/postgres/target/pulsar-io-debezium-postgres-${project.version}.nar</source></file>
```

Comment:
nit: alignment issue: use `space` instead of `tab`. 
```suggestion
    <file><source>${basedir}/../../pulsar-io/debezium/postgres/target/pulsar-io-debezium-postgres-${project.version}.nar</source></file>
```
use space instead of tab to make it align

Reply:
done: tab replaced with spaces [28037a7](https://github.com/apache/pulsar/commit/28037a77f0a1a28500812d1081fc2699e2cd1a8d)